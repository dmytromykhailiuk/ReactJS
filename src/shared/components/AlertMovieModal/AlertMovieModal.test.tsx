import { shallow } from "enzyme";
import { AlertMovieModal } from "./";
import React from "react";
import toJson from 'enzyme-to-json';

describe("AlertMovieModal", () => {
  it("shoud match first snepshot", () => {
    const wrapper = shallow(
      <AlertMovieModal 
        onCloseModal={() => {}}
        isSuccessAlert={true}
        alertMessage={"alert message"}
      />
    );
    expect(toJson(wrapper)).toMatchSnapshot("first");
  });

  it("shoud match second snepshot", () => {
    const wrapper = shallow(
      <AlertMovieModal 
        onCloseModal={() => {}}
        isSuccessAlert={false}
        alertMessage={"alert message"}
      />
    )
    expect(toJson(wrapper)).toMatchSnapshot("second");
  })
})