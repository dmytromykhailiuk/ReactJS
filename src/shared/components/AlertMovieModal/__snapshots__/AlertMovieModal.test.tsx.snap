// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AlertMovieModal shoud match first snepshot: first 1`] = `
<ModalWrapper
  onCloseModal={[Function]}
>
  <div
    className="alert-movie-modal__logo"
  >
    <CheckboxIcon
      isLarge={true}
    />
  </div>
  <div
    className="alert-movie-modal__title"
  >
    CONGRADULATIONS !
  </div>
  <div
    className="alert-movie-modal__message"
  >
    alert message
  </div>
</ModalWrapper>
`;

exports[`AlertMovieModal shoud match second snepshot: second 1`] = `
<ModalWrapper
  onCloseModal={[Function]}
>
  <div
    className="alert-movie-modal__logo"
  >
    !
  </div>
  <div
    className="alert-movie-modal__title"
  >
    SOMETHING WENT WRONG !
  </div>
  <div
    className="alert-movie-modal__message"
  >
    alert message
  </div>
</ModalWrapper>
`;
